pandas/core/generic.py

```diff
@@ -3137,6 +3137,78 @@ def clip_lower(self, threshold, axis=None):
         subset = self.ge(threshold, axis=axis) | isnull(self)
         return self.where(subset, threshold, axis=axis)
 
```

Comment:
this entire routine should be in ``pandas/core/frame.py`` as its only meant to work on a ``DataFrame`` right?